### PR TITLE
Migrate terra progress bar

### LINKED ISSUE #67: Migrate terra-progress-bar into repo
https://github.com/cerner/terra-progress-bar

### PR DESCRIPTION
**Pull request by bjankord**
Resolves #67

### CODE REVIEW

File: packages/terra-progress-bar/src/ProgressBar.jsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import './ProgressBar.scss';
+
+const propTypes = {
+  /**
+   * The color of the progress bar. Accepts a CSS color value.
```
Comment (Matt-Butler) [inline packages/terra-progress-bar/src/ProgressBar.jsx]: nit: remove css

File: packages/terra-progress-bar/src/ProgressBar.jsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  ]);
+
+  const normalizedValue = (value / max) * 100;
+
+  return (
+    <progress
+      {...customProps}
+      style={{ color: customProps.color }}
+      className={classes}
+      max={100}
```
Comment (Matt-Butler) [inline packages/terra-progress-bar/src/ProgressBar.jsx]: Is max always 100? This seems wrong if max is a prop

Comment (Matt-Butler) [inline packages/terra-progress-bar/src/ProgressBar.jsx]: Oh nevermind, I see on line 43 that the value is normalized.

**Comment by sid141288** [comment]
lgtm +1

**Pull request merged**